parsl/providers/local/local.py

```diff
@@ -206,7 +206,7 @@ def submit(self, command, tasks_per_node, job_name="parsl.localprovider"):
         script_path = "{0}/{1}.sh".format(self.script_dir, job_name)
         script_path = os.path.abspath(script_path)
 
-        wrap_command = self.worker_init + '\n' + self.launcher(command, tasks_per_node, self.nodes_per_block)
+        wrap_command = self.worker_init + f'\nJOBNAME=${job_name}\n' + self.launcher(command, tasks_per_node, self.nodes_per_block)
```

Comment:
the other providers `export` this - that means it will be visible to other processes. Maybe that doesn't affect things in this particular case but it is more consistent.

Reply:
Sure, I'll make the change.